tasks/styles.js

```diff
@@ -23,7 +23,21 @@ module.exports = (gulp, $, pkg) => {
   const cleanCssPluginOptions = {
     level: {
       2: {
-        all: true,
+        all: false,
+        mergeAdjacentRules: true, // controls adjacent rules merging; defaults to true
+        mergeIntoShorthands: true, // controls merging properties into shorthands; defaults to true
+        mergeMedia: true, // controls `@media` merging; defaults to true
+        mergeNonAdjacentRules: true, // controls non-adjacent rule merging; defaults to true
+        mergeSemantically: false, // controls semantic merging; defaults to false
+        overrideProperties: true, // controls property overriding based on understandability; defaults to true
+        removeEmpty: true, // controls removing empty rules and nested blocks; defaults to `true`
+        reduceNonAdjacentRules: true, // controls non-adjacent rule reducing; defaults to true
+        removeDuplicateFontRules: true, // controls duplicate `@font-face` removing; defaults to true
+        removeDuplicateMediaBlocks: true, // controls duplicate `@media` removing; defaults to true
+        removeDuplicateRules: true, // controls duplicate rules removing; defaults to true
+        removeUnusedAtRules: false, // controls unused at rule removing; defaults to false (available since 4.1.0)
+        restructureRules: false, // controls rule restructuring; defaults to false
+        skipProperties: [] // controls which properties won't be optimized, defaults to `[]` which means all will be optimized (since 4.1.0)
       }
     }
```

Comment:
I created a parallel [PR](https://jira.stibodx.com/stash/projects/BNNCUE/repos/bnncue-front-publication/pull-requests/1387/overview) in your repo to disable clean-css for the `@font-face` definitions in `base/_fonts.scss`, which should also fix the concrete issue.

However, it is true that we're enabling options that are not enabled by default with `all: true` currently. Based on the clean-css docs I think we should rather just use the defaults and remove the hashmap altogether:
```suggestion
    level: 2,
```
See https://github.com/jakubpawlowicz/clean-css#optimization-levels

@wrux Do you agree?

Reply:
I experienced this issue in the past on another site. If I remember correctly, this is caused by building stylesheets individually and cleancss assumes that the font face is not used elsewhere. 

This change to `gulp-task-collection` might have un-wanted side affects. It's probably easier to just go with the `/* clean-css ignore:start */` directives where relevant.

Another solution could be to create a `cleancss.config.js` file in each project, load the configuration and merge with the default cleancss config?